CHANGES.md

```diff
@@ -1,5 +1,9 @@
 # Changelog
 
+## 1.1.3, not published
+
+- Require Java 11 for Plugin Verifier 1.260+
```

Comment:
-> `runPluginVerifier`: verify required Java 11 environment for Plugin Verifier 1.260+